### ISSUE TITLE
The alt text for images is not update when we update the RSS feed

### ISSUE THREAD

**Issue by LOBsTerr** [LOBsTerr/newsroom_connector#10]
1) Add item with image
2) Update alt text in RSS feed
3) Reimport the item

ALT text should be correctly updated
